### PR TITLE
Message randomization 2

### LINKED ISSUE #26: Message randomization
- Convert multiple messages into an external file

### PR DESCRIPTION
**Pull request by kamontia**
@chaspy 
I implemented following feature:
- Display emoji randomly when `/kiemtaro` command is posted
- Determine the seed value by using system time

Close #26

### CODE REVIEW

**Comment by kamontia** [comment]
@chaspy I force pushed. I added some commits unrelated to the issue, but these fix is a bit, so it is included in this PR. Please review and merge

**Comment by kamontia** [comment]
@chaspy Hi, I processed rebase master. Please review and merge.

File: config/config.ini [hunk 1]
```diff
@@ -15,8 +15,8 @@ emoji1 = ":confetti_ball:"
          ":beers:"
          ":tada:"
          ":lying_face:"
-error_message1 = "なんやせっかちやな！リストは空っぽやで"
-                 "冗談は顔だけにしとき！リストは空っぽやで"
+error_message1 = "?! なんやせっかちやな！リストは空っぽやで"
```
Comment (chaspy) [inline config/config.ini:18]: What is the change for？

Comment (kamontia) [reply config/config.ini:18]: @chaspy Kimetaro says "むむっ、これや！" or "神のお告げに従って決めたったで" just before although list is empty. So kimetaro should do 'Nori-Tukkomi'. That's Japanese style.

**Review by kamontia — COMMENTED** [review]
I posted comment. Please check.

**Comment by kamontia** [comment]
@chaspy I divided commits and pushed.

**Review by chaspy — APPROVED** [review]
LGTM :+1

**Pull request merged**